hazelcast/src/main/resources/release_notes.txt

```diff
@@ -1,5 +1,23 @@
 This document lists the new features, enhancements, fixed issues and, removed or deprecated features for Hazelcast IMDG 3.12.x release. The numbers in the square brackets refer to the issues in Hazelcast's GitHub repositories.
 
+==== 3.12.2 - VISA8 - MIGRATION ====
+Enhancements
+
+* Introduced WAN throttling mechanism: this prevents the WAN consumers
+from getting overloaded by the WAN producers if transferring the WAN events
+takes less time than processing them in the target cluster. This can be the case
+with WAN synchronization if the network latency is low enough. It can be configured
```

Comment:
> ...if the network latency is low enough...

maybe we can add
> ...or if the source cluster is big

Every source cluster member replicates to every target member (it knows of), so the events piling up in the queues of the target members can get overwhelmed quickly in case of mass events, like sync.

Reply:
thanks, added in https://github.com/hazelcast/hazelcast/pull/24325/commits/c7b57e12193c37da1debec44678a1ac4bbab5ae1